### PR TITLE
Feature/about page

### PR DESCRIPTION
**Pull request by ethanresnick**
Changes:

- Create an InnerPage component that can be used as a template for about us, privacy policy, and other small inner pages.

- Pull the Intro component out to the top level, rather than having it a sub component of the HowItWorks component, so it can be reused on inner pages (like About).

- Delete the Contact nav link

@data-doge Please do review relatively soon, just so we don't get more merge conflicts :)

### CODE REVIEW

File: components/inner-page/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    return (
+      <div className="page">
+        <Header />
+          {this.props.children}
+        <Footer />
+      </div>
+    )
+  }
+}
+
```
Comment (data-doge) [inline components/inner-page/index.js]: nice

File: components/how-it-works-page/index.js [hunk 2]
```diff
@@ -1,17 +1,28 @@
 import React, { Component } from 'react'
 import Header from './../header'
 import Footer from './../footer'
-import Intro from './intro'
+import Intro from './../intro'
```
Comment (data-doge) [inline components/how-it-works-page/index.js]: yee good idea

File: components/intro/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+class Intro extends Component {
+  render () {
+    return (
+      <div className="intro">
+        <h1 className="intro__header">{this.props.heading}</h1>
+
+        {renderIf(this.props.subheading) (
+          <h2 className="intro__subheader">{this.props.subheading}</h2>
+        )}
```
Comment (data-doge) [inline components/intro/index.js]: added `renderIf` here

File: components/inner-page/index.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import Footer from './../footer'
 class InnerPage extends Component {
   render () {
     return (
-      <div className="page">
+      <div className="inner-page">
```
Comment (data-doge) [inline components/inner-page/index.js]: sry i'm just being anal 🌴

File: components/intro/index.js (hunk 3 above)
Comment (ethanresnick) [inline components/intro/index.js]: +1 Good to know

**Pull request merged**